--- ./src/main/java/io/github/dsheirer/audio/broadcast/AbstractAudioBroadcaster.java
+++ ./src/main/java/io/github/dsheirer/audio/broadcast/AbstractAudioBroadcaster.java
@@ -43,6 +43,7 @@
     protected int mStreamedAudioCount = 0;
     protected int mErrorAudioCount = 0;
     protected int mAgedOffAudioCount = 0;
+    protected long mLastStreamedTime = 0;

     /**
      * Constructs an instance
@@ -192,6 +193,8 @@
     public void incrementStreamedAudioCount()
     {
         mStreamedAudioCount++;
+        mLastStreamedTime = System.currentTimeMillis();
+        broadcast(new BroadcastEvent(this, BroadcastEvent.Event.BROADCASTER_LAST_STREAMED_TIME_CHANGE));
     }

     /**
@@ -218,6 +221,11 @@
         return mErrorAudioCount;
     }

+    public long getLastStreamedTime()
+    {
+        return mLastStreamedTime;
+    }
+
     /**
      * Name for this broadcaster thread
      */
